<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>601</width>
     <height>441</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>URW Gothic L</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="12" column="0">
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>URW Gothic L</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Email:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>URW Gothic L</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>License Number:</string>
      </property>
     </widget>
    </item>
    <item row="14" column="0">
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>URW Gothic L</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Notes:</string>
      </property>
     </widget>
    </item>
    <item row="14" column="1">
     <widget class="QLabel" name="labelNotes">
      <property name="font">
       <font>
        <family>URW Gothic L</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="15" column="1">
     <widget class="QPushButton" name="pushOk">
      <property name="font">
       <font>
        <family>URW Gothic L</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Okay</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="labelLicenseNum">
      <property name="font">
       <font>
        <family>URW Gothic L</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>License Number:</string>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QLabel" name="labelEmail">
      <property name="font">
       <font>
        <family>URW Gothic L</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Email:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>URW Gothic L</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Company Name:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="labelCompany">
      <property name="font">
       <font>
        <family>URW Gothic L</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Company Name:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>URW Gothic L</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Contact Name:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="labelContactName">
      <property name="font">
       <font>
        <family>URW Gothic L</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Contact Name:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>URW Gothic L</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Contact Number:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="labelContactNum">
      <property name="font">
       <font>
        <family>URW Gothic L</family>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Contact Number:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
